packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/about.a.tool.mdx

```diff
@@ -1,123 +1,73 @@
-import { Badge } from 'terra-dev-site/package.json?dev-site-package';
+import { Badge } from '@cerner/terra-dev-site/package.json?dev-site-package';
+import { Notice } from '@cerner/terra-docs';
 
 <Badge />
 
 # Terra Dev Site
 
-Terra-dev-site offers a quick test site to host examples and documentation for your react components or markdown documentation. Terra dev site is intended for speed development of reusable node modules utilizing terra that are intended to be shared between applications.
+Terra-dev-site is a webpack plugin for generating a documentation and testing site for content built with the terra application framework.
```

Comment:
We refer to the package contents in a couple different ways throughout this file:
Terra-dev-site 
terra-dev-site 
Terra dev site
"the dev site"

Can we just use TerraDevSite as a reference point throught throughout?

```suggestion
TerraDevSite is a webpack plugin for generating a documentation and testing site for content built with the terra application framework.
```

Reply:
I will fix this throughout this file.